### ISSUE TITLE
[Feature Request]: Add additional LDAP sync fields

### ISSUE THREAD

**Issue by YannickJaeckle** [grokability/snipe-it#11400]
### Is your feature request related to a problem? Please describe.

Is it possible to add more AD attributes like streetAddress, wWWHomePage, postalCode, l (AD attribute for location) for ldap sync? 
Currently we change it manuelly per user. It would be great if this could expand to more LDAP attributs. 

![image](https://user-images.githubusercontent.com/82890374/175953628-886a57d6-9207-4aa1-9476-aea80ddababb.png)

Snipeit Version : 6.0.5

### Describe the solution you'd like

Could be an optional field for sync these attributes.

**Comment by nghia-dang** [comment]
I actually use the Location LDAP Search OU to pretty much emulate a custom attribute by using location as a fake attribute... It would be nice to be able to choose to add a group of custom attributes that we can self label instead of people just constantly asking for specific ones.

**Comment by valireality** [comment]
It will be great also profile picture to be synced from LDAP/AD.

Thanks

**Comment by YannickJaeckle** [comment]
Are there some news about this feature request available? :-)

**Comment by zbrand55** [comment]
> 

How do you do this exactly? I'm trying to set locations for users that are imported from LDAP.

**Comment by viperthunder** [comment]
Yes, this would be awesome! There are quite a few ldap attributes missing from the GUI . City, Zip code, street address.. This leaves a lot of info to be manually entered each time we import a new user

**Comment by snipe** [comment]
https://github.com/grokability/snipe-it/pull/17650
